### PR TITLE
Use AppEngine flex env with 3G memory

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: cloudbuild.deploy.yaml [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+  - id: deploy
+    name: "gcr.io/cloud-builders/gcloud"
+    entrypoint: /bin/bash
+    args:
+      - -c
+      - |
+        set -x
+        cd server
+        gcloud config set project datcom-browser-staging
+        # bdist_wheel for grpcio takes a long time, seeting timeout to be 1200s
```
Comment (beets) [inline cloudbuild.deploy.yaml:29]: nit: s/seeting/setting

what happens if we hit the timeout? should we continue with the deployment?

File: server/app_prod.yaml [hunk 2]
```diff
@@ -12,11 +12,19 @@
...
+entrypoint: gunicorn -b :$PORT main:app
 
-instance_class: F4_1G
-automatic_scaling:
-  min_instances: 3
+runtime_config:
+  python_version: 3
+
+manual_scaling:
+  instances: 10
```
Comment (beets) [inline server/app_prod.yaml:23]: should we get more instances for prod vs staging?

File: cloudbuild.deploy.yaml (hunk 1 above)
Comment (shifucun) [reply cloudbuild.deploy.yaml]: Then cloudbuild would error out. Tried locally it the gcloud build takes < 10min, so this should be safe. Will need to watch for the first few times deployment

File: server/app_prod.yaml (hunk 2 above)
Comment (shifucun) [reply server/app_prod.yaml:23]: Yeah. I don't know a good number to set for now. Might let it run for a while and adjust if we saw performance issues?

Comment (beets) [reply server/app_prod.yaml:23]: ok

File: cloudbuild.deploy.yaml (hunk 1 above)
Comment (beets) [reply cloudbuild.deploy.yaml]: ok, let's keep an eye on it

**Pull request merged**